tests/utils/streams/test_kafka.py

```diff
@@ -32,7 +46,7 @@ def decode(self, value: KafkaPayload) -> int:
         return int(value.value.decode("utf-8"))
 
 
-class KafkaStreamsTestCase(StreamsTestMixin, TestCase):
+class KafkaStreamsTestCase(StreamsTestMixin[int], TestCase):
```

Comment:
This will become `StreamsTestMixin[KafkaPayload]` when the codec is removed from `KafkaConsumer`/`KafkaProducer`.

Reply:
Went ahead and did this with fc647585ab13723ab99ba0a18cb38d21f0f1ab3c so the follow up change(s) will only be removing the codec and the type variable.